modules/tutorials/pages/gsql-101/parameterized-gsql-query.adoc

```diff
@@ -18,10 +18,12 @@ Now, let's write our first GSQL query. We'll display all the direct (1-hop) neig
 [source,gsql]
 ----
 USE GRAPH social
+SET syntax_version="v2"
```

Comment:
Just being curious: do we need to explicitly set the version as v2 while it's already the default version?